src/Ampersand/Classes/Relational.hs

```diff
@@ -110,10 +109,6 @@ instance Relational Expression where -- TODO: see if we can find more property c
       EKl1 e -> isFalse e
       EFlp e -> isFalse e
       ECpl e -> isTrue e
-      EDcD {} -> False
-      EDcI {} -> False
-      EEps {} -> False
-      EDcV {} -> False
```

Comment:
It is dangerous to get rid of these specifics. I would rather get rid of the `_ -> False` part, so when an extra constructor is added some time in the future, we are forced to think in over here.